modules/querying/pages/syntax-versions.adoc

```diff
@@ -1,9 +1,73 @@
 = Query Language Syntax Versions
 :description: Summary of the differences between GSQL syntax V1 and V2.
 
-GSQL has two syntax versions: V2 and V1 (legacy version).
+GSQL has three syntax versions: V3, V2, and V1 (legacy version).
 In TigerGraph 3.5, the default syntax version was changed to V2.
-This page provides a summary of the differences between V1 and V2, as well as guidance on how to change your V1 query into a V2 query.
+Currently, V2 remains the default syntax version. In a future release, V3 will become the default syntax version.
+This page provides a summary of the differences between V2 and V3, as well as V1 and V2. It also includes guidance on how to change your V1 query into a V2 query.
+
+== Differences between V2 and V3
+
+The table below highlights the differences between V2 and V3, focusing on the `FROM` clause.
+
+|===
+|Feature|V2 |V3
+
+|Vertex-Edge-Vertex Markers
+|Use `source_vset -(edge_set)- target_vset` notation.
+|Use `(source_vset) -[edge_set]- (target_vset)` notation, improving clarity and aligning with GQL style.
+
+|Alias Order
+|Object sets and aliases follow `source_vset:S -(edge_set:E)- target_set:T`.
+|Object sets and aliases follow `(S:source_vset) -[E:edge_set]- (T:target_set)`, making alias assignment explicit and intuitive.
+
+|Symbols for Edge Direction
+|Arrowheads `<` or `>` are placed inside parentheses after the edge type to indicate direction (e.g., `-(EdgeType:e)-`).
+|Same as V2, but notation uses square brackets for edges, such as `[edge_set]` for improved clarity.
```

Comment:
This section needs to be expanded.
We don't have a great summary of V2 notation. See https://docs.tigergraph.com/gsql-ref/4.1/querying/select-statement/#_path_edge_pattern

For V3, see https://github.com/tigergraph/ecosys/blob/master/demos/guru_scripts/docker/tutorial/4.x/README.md#pattern-summary

Note the section on repeated hops. That is another difference that I forgot to mention.